src/store/index.ts

```diff
@@ -1,41 +1,67 @@
 import { createStore, AnyAction } from 'redux';
 import { composeWithDevTools } from 'redux-devtools-extension';
+import {
+  SET_TODOS, REMOVE_TODO, SET_LOADED, SET_LOADING, SET_SORT_TYPE, REVERSE,
+} from './constants';
 
 // Action types - is just a constant. MUST have a unique value.
-const START_LOADING = 'START_LOADING';
-const FINISH_LOADING = 'FINISH_LOADING';
+// const START_LOADING = 'START_LOADING';
+// const FINISH_LOADING = 'FINISH_LOADING';
 
 // Action creators - a function returning an action object
-export const startLoading = () => ({ type: START_LOADING });
-export const finishLoading = (message = 'No message') => ({ type: FINISH_LOADING, message });
+// export const startLoading = () => ({ type: START_LOADING });
+// export const finishLoading = (message = 'No message') => ({ type: FINISH_LOADING, message });
+const initialState: RootState = {
+  todos: [],
+  isLoading: false,
+  isLoaded: false,
+  sortType: '',
+  isReverse: false,
+};
 
 // Selectors - a function receiving Redux state and returning some data from it
-export const isLoading = (state: RootState) => state.loading;
-export const getMessage = (state: RootState) => state.message;
+// export const isLoading = (state: RootState) => state.loading;
+// export const getMessage = (state: RootState) => state.message;
+export const setTodos = (todos: Todo[]) => ({ type: SET_TODOS, todos });
+export const deleteTodo = (id: number) => ({ type: REMOVE_TODO, id });
+export const isLoading = () => ({ type: SET_LOADING });
```

Comment:
if you set isLoading state, then the name for this function should  be `setIsLoading` or `setLoadingState`